user-journeys/application_developer-foundational.html

```diff
@@ -16,80 +16,112 @@
+-->
+<br><br> 
+
+        <ul>
+          <li>Minikube built-in Docker daemon (do we want to mention rkt support? I'm inclined not to do so, but it's currently on the Minikube GSG page)</li>
+          <li>minikube start</li>
+          <li></li>
+        </ul>
 <br><br>
-        Aenean suscipit arcu ac leo tincidunt tempus. Donec maximus tellus libero, ac ullamcorper magna lobortis ac. Integer mollis nisl vitae magna gravida, nec ornare ex consectetur. Sed mattis tincidunt nisi, at consequat tellus malesuada non. Integer vel semper nisi, ut fringilla velit. Nam felis ex, congue non dui vitae, sollicitudin convallis turpis. Phasellus porttitor maximus turpis, in varius nibh fermentum aliquam. Cras finibus lacus non diam porttitor porttitor. Nulla fringilla sagittis nibh nec condimentum. Duis egestas mauris nec dolor hendrerit ullamcorper sit amet in mi. Phasellus sollicitudin justo diam.
+        Access the cluster/Get basic information about the cluster: <code>kubectl get nodes</code> Drill down further? Explain what nodes do a little bit here?
+<br><br>
+        <a href="https://kubernetes.io/docs/tutorials/kubernetes-basics/explore-intro/">Viewing Pods and Nodes</a>
+<br><br>
+        Install a sample application. Recommendations? 
       </div>
 
       <div class="topheader" id="2">
          <span class="numberCircle"><span>2</span></span></i>&nbsp;&nbsp;Deploy, scale, and update an application
       </div>
 
       <div class="docsection1">
-        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean id felis non enim faucibus lacinia. Aliquam massa mauris, interdum a ex ut, sagittis rutrum nulla. In pellentesque est at molestie fringilla. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Donec eu mi at velit lacinia venenatis ac nec sem. In volutpat pellentesque dui ut commodo. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Duis sollicitudin eleifend felis non facilisis. Pellentesque leo urna, congue id auctor non, varius a nunc. Duis ultrices, odio ut hendrerit suscipit, nisi mauris dignissim mauris, nec bibendum ante neque ut augue. Interdum et malesuada fames ac ante ipsum primis in faucibus. Nam condimentum libero sit amet rutrum fermentum. Proin posuere condimentum odio. Aenean volutpat, ex vel tincidunt tincidunt, massa odio condimentum lectus, vel iaculis libero dolor in nisi.
+        To deploy and manage an application in Kubernetes, you write a manifest, also called a configuration or config file. The manifest is written in YAML or JSON, and describes the desired state of the application and related resources as Kubernetes should maintain them. (if this content stays, needs to be moved)
```

Comment:
It might be misleading to say you write a single manifest to describe an entire application. You'd probably have several manifests.

We could steer clear of the general case and say something like:
You can deploy a simple application by using a Deployment manifest.